Automattic/sensei
# Lesson Admin: Populate modules on course change

This commit adds an enhancement to the lesson admin area.
It autopopulates the module select box when a user selects
a course.

It introduces the following usability benefits:

* I can create a new lesson and immediately set the module
  after selecting a course
* I can edit an existing lesson and immediately update the module,
  if I update the course

Note: any time we change a lesson's course, any selected
module gets deselected. Order postmeta is preserved.

Closes #1309 

To Test:

* Create a course and 2 modules for it, as well as one without modules
* Create a new lesson. Select the course with modules. After a moment, the module select box should show. Select the course without modules. No modules show.
